catroid/src/main/java/org/catrobat/catroid/ui/ScratchConverterActivity.java

```diff
@@ -272,7 +249,7 @@ public void onActivityResult(int requestCode, int resultCode, Intent data) {
 			List<String> results = data.getStringArrayListExtra(
 					RecognizerIntent.EXTRA_RESULTS);
 			String spokenText = results.get(0);
```

Comment:
just call it directly as data.getStringArrayListExtra(...).get(0) no need to create a list to buffer the return value.